lib/src/Uncertainty/StatTests/openturns/FittingTest.hxx

```diff
@@ -79,8 +79,9 @@ public:
                     const UnsignedInteger estimatedParameters = 0);
 
   /** Bayesian Information Criterion computation */
-  static Scalar BIC(const Sample & sample,
-                    const DistributionFactory & factory);
+  static Distribution BIC(const Sample & sample,
+                          const DistributionFactory & factory,
+                          Scalar & bestBICOut);
```

Comment:
Wait, if you swap the arguments you have the distribution first:
```
estimated_distribution, bic = FittingTest.BIC(sample, factory)
```
whereas kolmogorov returns the test result first:
```
test_result, estimated_distribution = FittingTest.Kolmogorov(sample, factory)
```

Reply:
indeed.

Reply:
BIc is a `scalar` wheras `test_result` is of type `TestResult`
I suggest to get `estimated_distribution,  bic` and `estimated_distribution,   test_result` 

ok for you?

Reply:
Yes, my remark was just the order of the distribution

Reply:
You can chose the order of the most used convention.